src/main/kotlin/skytils/skytilsmod/core/Config.kt

```diff
@@ -2618,6 +2618,22 @@ object Config : Vigilant(
     )
     var compactHider = 0
 
+    @Property(
+        type = PropertyType.SELECTOR, name = "Pristine Hider",
+        description = "Removes Pristine messages from mining.",
+        category = "Spam", subcategory = "Miscellaneous",
+        options = ["Normal", "Hidden", "Separate GUI"]
+    )
+    var pristineHider = 0
```

Comment:
I think there already is a pristine hider I'm not too sure

Reply:
I looked through ingame and through the code and could not find it. I might have missed it, though.

Reply:
https://github.com/Skytils/SkytilsMod-Data/blob/main/SpamFilters.json#L14-L19

Reply:
@Sychic Gotcha. Is there a reason why some stuff, like that, is within the Spam Filters system from the Skytils-Data repo and the Spam Filters menu ingame, rather than the general spam config section? 

From a user standpoint, seeing compact be in the spam section and not pristine was confusing, leading me to think that it was not an option. I generally thought of the spam filters for custom things, rather than gameplay features, though that, in your team's mind, might not be the intended usecase.

Reply:
The main reason for creating the repo filters was so then we could roll out new spam hiders remotely without having to release a new version of the mod. Though the custom spam hiders has mostly overtaken this usage since anyone can create any spam hider they want.

Reply:
Gotcha. I am fine not having it added, if that is your guy's preference (and I can remove it from this PR), but I would also think that it would fit with compact in the actual menu. 

I do see the desire for adding things without a new version of the mod, I maybe would add a link to the spam filters in the spam section then, as it was not apparent to me that it would be in that section, as well as talking to a few friends. Just seems like some features will get missed by users. 

In my opinion, it is worth having most of the general spam features in the actual config, even if that requires a new mod version, and having the spam filters section mostly be custom ones, especially since Skytils has such a great updating system. But of course, that is just my opinion, feel free to do whatever you think is best for the vision of the mod.

Reply:
Looking again -- I do see "Legacy Hiders" inside the Spam Filters menu. Maybe at some point Spam migrates entirely to the Spam Filter system, including allowing custom, so there is not options duplicated in two menus.

Reply:
> Looking again -- I do see "Legacy Hiders" inside the Spam Filters menu. Maybe at some point Spam migrates entirely to the Spam Filter system, including allowing custom, so there is not options duplicated in two menus.

That would be the goal but waiting until we have search for it, the only real reason for a legacy hider now is for toasts (as there's no good way to custom add it)